flink-ml-lib/src/main/java/org/apache/flink/ml/recommendation/swing/Swing.java

```diff
@@ -147,9 +148,11 @@ public Table[] transform(Table... inputs) {
                                 new ComputingSimilarItems(
                                         getK(),
                                         getMaxUserNumPerItem(),
+                                        getMaxUserBehavior(),
```

Comment:
nit:  `getMaxUserBehavior` -> `getMaxUserBehaviorNum`, which is more consistent with other setters like `getMaxUserNumPerItem`. Same as `getMinUserBehavior`.

Reply:
After offlinediscussion, we agree to keep the current parameter name to be consistent with the official Swing implementation.